packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/saveEditInfo.ts

```diff
@@ -0,0 +1,13 @@
+import ImageEditInfo, { IMAGE_EDIT_INFO_NAME } from '../types/ImageEditInfo';
+
+/**
+ * @internal
+ * Save edit info to image
+ * @param image The image to save edit info to
+ * @param editInfo The edit info to save
+ */
+export default function saveEditInfo(image: HTMLImageElement, editInfo: ImageEditInfo) {
+    if (image) {
+        image.dataset[IMAGE_EDIT_INFO_NAME] = JSON.stringify(editInfo);
```

Comment:
could we possibly use object url here as editInfo could be extremely long data, maybe we can do that later?

Reply:
We have EditImage event which will give the flexibility for application to relace dataUrl to be ObjectUrl